scripts/Tools/Makefile

```diff
@@ -612,17 +612,25 @@ endif
 # Drive configure scripts for support libraries (mct)
 #------------------------------------------------------------------------------
 
+ifeq ($(CIME_MODEL),cesm)
+    #This will be changed to EXTERN_PATH = $(SRCROOT)/libraries when the new
+    #share repos are used for cesm2.
+    EXTERN_PATH = $(CIMEROOT)/src/externals
```

Comment:
Is this ultimately going to be moved to config_files.xml and set there?

Reply:
My understanding is once we switch to nuopc the mct build will go away.  We will also switch to using only pio2.  At that time I can switch the pio2 path to $SRCROOT/libraries.  This is assuming the e3sm isn't going to be using pio2.  So EXTERN_PATH isn't going to move to config_files.xml, it's just going to go away.